tests/table_test.go

```diff
@@ -1001,6 +1002,7 @@ Repositories with a successful run:
 			verify: func(t *testing.T, vcMock *vcmock.VersionController, runData runData) {
 				require.Len(t, vcMock.PullRequests, 1)
 				assert.True(t, fileExist(t, workingDir, "tmp-test"))
+				assert.Contains(t, runData.logOut, multigitter.NormalizePath(filepath.Join(workingDir, "tmp-test/")))
```

Comment:
Just checking where the logs claim the repo is cloned to will not guarantee that it will actually use that directory.

```go
tmpDir, err := createTempDir(r.CloneDir)
log.Debug("Cloning into directory ", NormalizePath(tmpDir))
CloneAndRun(os.TempDir())
```
⬆️ Pseudocode that would pass the test.